### DISCUSSION TITLE
Accessing SocOn via Web for management

### DISCUSSION THREAD

**Discussion by hamads007** [Security-Onion-Solutions/securityonion#9817] [General]
Hi,
I have downloaded SO on VMWare Player for my final year project,  
I have got confused about which IP address to use to access secOn vis  web 
shall I go to my browser on VMware to access it using its VMware network range 192.168.22.0/24 or go to my main browser on my pc to access it, please?  
Please help I have spent my hours installing it a few times in different mode and I can not access it 
Thanks

**Comment by cm-ops** [comment]
What IP/range did you allow access to the SOC? You can check with `so-allow-view.`  If you are able to ping the VM from your workstation, then check your IP and run `so-allow`, select `a`, and add your `IP`. Then try to get to SOC.

**Reply by hamads007** [reply]
Hi CM-OPS
IP address is from the lan range of the workstation for example 192.168.111.100 ( Lan network adaptor) 
I have added a second network adaptor for my testing lab, range 192.168.2.0/24
so I went to the Ubuntu web browser and tried 192.168.111.100, but nothing opened
or shall I have IP add from the (host only range) from example 192.168.2.100
Also, shell I open SO from my pc web browser or my ubuntu web browser from the workstation
Thank you so much for your advise, 
Regards

**Reply by cm-ops** [reply]
What is the IP of your PC? Are you able to ping SO from your PC? If yes, run `so-allow` and add your PC IP. Then you should be able to open a web browser on your PC and get to SOC.
